.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - "*"
+
+jobs:
+  antora:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.ref, 'tags') }}
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Configure Git
+      run: |
+        git config user.name "GitHub Actions"
+        git config user.email "actions@github.com"
```

Comment:
Maybe this could be "Antora via GitHub Action" or alike? I would put at least some information here that makes the commits that result of this action relatable to this action.